src/common/playlists/PlayList.jsx

```diff
@@ -27,7 +36,7 @@ const PlayList = () => {
       if (parsedQuery) {
         translatedQuery = QueryDBTranslator(parsedQuery);
       }
-      const res = await getPlaysByFilter(translatedQuery);
+      const res = await getPlaysByFilter(translatedQuery, sortBy);
```

Comment:
`sortBy` parameter is it being used anywhere inside getPlayByFilter function?

Reply:
Yes, it is being used.